app/features/tasks.py

```diff
@@ -44,7 +45,7 @@ def insert_tasklist(self, p_name):
 
         results = self.service.tasklists().insert(body=tasklist_details).execute()
         
-        print(u'{0} ({1})'.format(results['title'], results['id']))
+        self.tasklist.append(u'{0} ({1})'.format(results['title'], results['id']))
```

Comment:
why would we need to append to tasklist?, we already ask google to scrape for any tasks in the list_task function